src/retrieveMovies.js

```diff
@@ -0,0 +1,222 @@
+/*eslint-disable */
+
+import addComment from "../modules/addComment.js";
+import PopupComment from "../modules/PopupComment.js";
+
+const baseurl = "https://api.tvmaze.com/";
+const retrieveshows = async () => {
+
+  const request = await fetch("https://api.tvmaze.com/shows");
+  const response = await request.json();
+  return response;
+};
+
+const updateLikeCount = async (itemId, updatedCount) => {
+  const ID = "P8F6LlpZ9NxzdStT1SIa";
+  const baseUrl =
+    "https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/";
+  const url = `${baseUrl}apps/${ID}/likes`;
+
+  const requestOptions = {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json; charset=UTF-8",
+      "Access-Control-Allow-Origin": "http://localhost:9000",
+      "Access-Control-Allow-Methods": "POST",
+      "Access-Control-Allow-Headers": "Content-Type, Authorization",
+    },
+    body: JSON.stringify({ item_id: itemId }),
+  };
+
+  const response = await fetch(url, requestOptions);
+  return response;
+};
+
+const saveLikeCount = (itemId, count) => {
+  localStorage.setItem(itemId, count.toString());
+};
+
+const getLikeCount = async (itemId) => {
+  const ID = "P8F6LlpZ9NxzdStT1SIa";
```

Comment:
[REQUIRED] Kindly ensure that the methods handling each counter should have their own separate module

Reply:
resoleved and thanks